ui/apps/dashboard/src/components/FatalError.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import { useEffect } from 'react';
+import { Alert } from '@inngest/components/Alert';
+import { Button } from '@inngest/components/Button';
+import { Link } from '@inngest/components/Link';
+import * as Sentry from '@sentry/nextjs';
+
+import { pathCreator } from '@/utils/urls';
+
+type Props = {
+  error: Error & { digest?: string };
+  reset: () => void;
+};
+
+export function FatalError({ error, reset }: Props) {
+  useEffect(() => {
+    Sentry.captureException(error);
+  }, [error]);
+
+  return (
+    <div className="mt-32 flex w-full flex-col items-center justify-center gap-4">
+      <Alert severity="error">
+        <p className="mb-4 font-semibold">{error.message}</p>
+
+        <p>
+          An error occurred! Refresh the page to try again. If the problem persists, contact
+          support.
+        </p>
+      </Alert>
+
+      <div className="flex gap-4">
+        <Button btnAction={() => reset()} kind="primary" label="Try again" />
+
+        <Link href={pathCreator.support()}>Contact support</Link>
```

Comment:
Can we put this link inside the alert? With a `className="inline-flex"` it should align nicely with the text

Reply:
John wants the link outside the alert. He also wanted the button and link left-aligned so I made that change